core/src/main/resources/pl/messages.properties

```
@@ -1416,3 +1416,18 @@ coronastats.deaths.today=Zgonów dzisiaj:
 coronastats.critical=Krytycznych przypadków:
 coronastats.updated=Stan na
 coronastats.unknown.country=Nie ma takiego kraju! Przestań zmyślać kraje, użyj angielskiej nazwy lub skrótu ISO.
+
+ustawpoziom.help.description=Ustawia poziom uprawnień dla danej komendy
+ustawpoziom.help.uzycie=<list|set|remove> [komenda] [numeryczny poziom uprawnień]
+ustawpoziom.help.extended=\nPoziomy uprawnień znajdziesz pod komendą `permlevel`\nPrzykładowe użycie: \n```\nustawpoziom list\nustawpoziom set userinfo 2\nustawpoziom remove userinfo\n```
+ustawpoziom.list.desc=%s - %s (%s poziom)
```

chyba tak by bardziej pasowało
```suggestion
ustawpoziom.list.desc=%s - %s (poziom %s)
```